sdk/tests/conformance/extensions/khr-parallel-shader-compile.html

```diff
@@ -11,33 +11,70 @@
 <link rel="stylesheet" href="../../resources/js-test-style.css"/>
 <script src="../../js/js-test-pre.js"></script>
 <script src="../../js/webgl-test-utils.js"></script>
+<style>
+.spinner {
+    width: 100px;
+    height: 100px;
+    border: 20px solid transparent;
+    border-top: 20px solid black;
+    border-radius: 100%;
+    text-align: center;
+    padding: 10px;
+}
+@keyframes rotation {
+    from { transorm: rotate(0); }
+    to { transform: rotate(360deg); }
+}
+</style>
 </head>
 <body>
 <div id="description"></div>
+<button onclick='compileShaders()'>The spinners below should not stutter when you click this button.</button>
```

Comment:
Can you think of a way - like measuring requestAnimationFrame timestamps - to programmatically catch dropped frames over a period of time, rather than relying on manual testing?

Reply:
The test kind of does, by requiring several frames to elapse during the compilation time. But it can't verify that pixels on the screen actually change. That's what the manual test is for.

Reply:
I see. Thanks for clarifying this.